pyproject.toml

```diff
@@ -1,15 +1,15 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm"]
+requires = ["setuptools", "setuptools-scm", "requests", "numpy", "pandas", "lxml", "matplotlib"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "ifk-scb-compilations"
+name = "ifk_scb_compilations"
 version = "0.1.0"
 description = "Unspecified."
 license = { text = "MIT" }
 readme = "README.md"
 authors = [
-    { name = "Christoffer Strömberg", email = "info@ingenjorsarbeteforklimatet.se" },
+    { name = "IFK", email = "info@ingenjorsarbeteforklimatet.se" },
```

Comment:
Perhaps update the copier file with this information too.

Reply:
done.